pkgs/development/ocaml-modules/kqueue/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildDunePackage
+, dune-configurator
+, lib
+, fetchurl
+, ppx_expect
+, ppx_optcomp
+}:
+
+buildDunePackage rec {
+  pname = "kqueue";
+  version = "0.3.0";
+
+  minimalOCamlVersion = "4.12";
+
+  src = fetchurl {
+    url = "https://github.com/anuragsoni/kqueue-ml/releases/download/${version}/kqueue-${version}.tbz";
+    hash = "sha256-MKRCyN6q9euTEgHIhldGGH8FwuLblWYNG+SiCMWBP6Y=";
+  };
+
+  propagatedBuildInputs = [
+    dune-configurator
+    ppx_expect
+    ppx_optcomp
+  ];
```

Comment:
None of these libraries need to be propagated (i.e., use `buildInputs` in place of `propagatedBuildInputs`).

Moreover, `ppx_expect` belongs to `testInputs`.

Reply:
Sure, makes sense to me! Done.